examples/apis/invoice/get/main.go

```diff
@@ -0,0 +1,28 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/invoice"
+)
+
+func main() {
+	cfg, err := config.New("{{ACCESS_TOKEN}}")
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	client := invoice.NewClient(cfg)
+
+	invoiceID := "123"
+
+	inv, err := client.Get(context.Background(), invoiceID)
```

Comment:
aqui pode colocar completo
vamos tentar encurtar só quando muito necessário ou quando forem acrônimos ou quando for um nome comum (ex: res para respostas de APIs)
```suggestion
	invoice, err := client.Get(context.Background(), invoiceID)
```

Reply:
Perfeito! 
Gabs, fica dando um aviso "Variable 'invoice' collides with imported package name". Vamos seguir com esse nome mesmo?